doc/api_assets/style.css

```diff
@@ -1,5 +1,6 @@
 /*--------------------- Layout and Typography ----------------------------*/
 html {
+  -webkit-overflow-scrolling: touch;
```

Comment:
Hmmm, I think this may have negative effects on macs? we should probably restrict it to some size? (wish there was a better way :disappointed: )